backend/cinema_service/settings.py

```diff
@@ -175,3 +178,7 @@
     "REFRESH_TOKEN_LIFETIME": timedelta(days=1),
     "ROTATE_REFRESH_TOKENS": False,
 }
+
+CORS_ALLOWED_ORIGINS = [
+    "http://localhost:5173",
+]
```

Comment:
add a blank line

Reply:
But I added screenshots of all the pages with the working frontend, as specified in the condition

Reply:
then I added this comment accidentally, sorry